### PR TITLE
cc-input-text with tags: set default font back to `--cc-ff-monospace`

### LINKED ISSUE #510: cc-input-text (tags): change font-family back to monospace
# Context

Following #453, `cc-input-text` no longer uses the monospace font for tags.

# Goals

* We want the font to be monospace by default when using tags (**only when using tags**, it should still inherit in other cases).
* It should still be customizable by using the CSS custom prop.

### PR DESCRIPTION
**Pull request by florian-sanders-cc**
Fixes #510

### CODE REVIEW

**Review by roberttran-cc — APPROVED** [review]
LGTM! :+1:

**Review by pdesoyres-cc — APPROVED** [review]

**Comment by hsablonniere** [comment]
I update the JSDoc to specify that the default changed:

```
@cssprop {FontFamily} --cc-input-font-family - The font-family for the input content (defaults: `--cc-ff-monospace` from the them).
```

**Comment by hsablonniere** [comment]
@florian-sanders-cc I want to confirm with you wether this is a breaking change before merging.

**Comment by florian-sanders-cc** [comment]
> @florian-sanders-cc I want to confirm with you wether this is a breaking change before merging.

Did not think about that. I think it is indeed a breaking change :sweat_smile:.
I'll reword the commit and add it to the changelog :+1:

**Comment by florian-sanders-cc** [comment]
> I update the JSDoc to specify that the default changed:
> 
> ```
> @cssprop {FontFamily} --cc-input-font-family - The font-family for the input content (defaults: `--cc-ff-monospace` from the them).
> ```

thanks :wink:

File: src/atoms/cc-input-text.js [hunk 1]
```diff
@@ -50,7 +50,7 @@ function arrayEquals (a, b) {
  * @event {CustomEvent} cc-input-text:requestimplicitsubmit - Fires when enter key is pressed in simple mode, in tags mode or when ctrl+enter is pressed in multi mode.
  * @event {CustomEvent<string[]>} cc-input-text:tags - Fires an array of tags whenever the `value` changes (separated by spaces).
  *
- * @cssprop {FontFamily} --cc-input-font-family - The font-family for the input content (defaults: `inherit`).
+ * @cssprop {FontFamily} --cc-input-font-family - The font-family for the input content (defaults: `--cc-ff-monospace` from the them).
```
Comment (florian-sanders-cc) [inline src/atoms/cc-input-text.js:53]: @hsablonniere ahh it's a bit more complicated than this actually.
It only defaults to `--cc-ff-monospace` if you use tags. If not, it inherits.
I'll update the desc if that's ok with you ?

Comment (hsablonniere) [reply src/atoms/cc-input-text.js]: Nice :+1:

**Pull request merged**